Version 11.00 added multi-threading support, but we need to link to -lpthread
to resolve symbols.

Version 11.41 renamed IO.h to IAPEIO.h, but they didn't update CMakeLists.txt

--- CMakeLists.txt.orig	2025-09-01 19:26:38 UTC
+++ CMakeLists.txt
@@ -6,6 +6,8 @@ option(BUILD_UTIL "Build the mac command line tool" ON
 option(BUILD_SHARED "Build libMAC as a shared library" ON)
 option(BUILD_UTIL "Build the mac command line tool" ON)
 
+find_package(Threads)
+
 #
 # Determine interface version
 #
@@ -74,7 +76,7 @@ set(MAC_PUBLIC_HEADERS
         Shared/All.h
         Shared/APETag.h
         Shared/CharacterHelper.h
-        Shared/IO.h
+        Shared/IAPEIO.h
         Shared/MACDll.h
         Shared/MACLib.h
         Shared/NoWindows.h
@@ -91,8 +93,8 @@ set(MAC_SHARED_HEADERS
         Source/Shared/CPUFeatures.h
         Source/Shared/CRC.h
         Source/Shared/GlobalFunctions.h
+        Source/Shared/IAPEIO.h
         Source/Shared/ID3Genres.h
-        Source/Shared/IO.h
         Source/Shared/MACDll.h
         Source/Shared/MemoryIO.h
         Source/Shared/MFCGlobals.h
@@ -229,6 +231,7 @@ if(BUILD_SHARED)
 if(BUILD_SHARED)
     add_library(MAC SHARED $<TARGET_OBJECTS:shared> $<TARGET_OBJECTS:objects>)
     set_target_properties(MAC PROPERTIES VERSION ${MAC_SOVERSION} SOVERSION ${MAC_SOVERSION})
+    target_link_libraries(MAC Threads::Threads)
 
     if(WIN32)
         if(${ARCHITECTURE} STREQUAL x86)
@@ -315,7 +318,7 @@ if(BUILD_UTIL)
        Source/Console/Console.cpp)
 
     add_executable(macutil ${MAC_UTIL_HEADER_FILES} ${MAC_UTIL_SOURCE_FILES} $<TARGET_OBJECTS:shared>)
-    target_link_libraries(macutil MAC)
+    target_link_libraries(macutil MAC Threads::Threads)
 
     target_include_directories(macutil PUBLIC
        ${CMAKE_CURRENT_SOURCE_DIR}/Source/MACLib
